IFC4x3/Constants/h/HELMERTCURVE_14V3RUc6LFh9bZo1rEFhxL.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <DocConstant xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="HELMERTCURVE_14V3RUc6LFh9bZo1rEFhxL" Name="HELMERTCURVE" UniqueId="447c36de-9865-4fac-9963-c81d4e3ebed5">
 	<Documentation>Helmert transition.\
```

Comment:
What's the point of the backslash at the end? It seems out of place.

Reply:
It isn't used in any of the other changed descriptions in this PR. I'd prefer to see consistency, but can leave it as is.

Reply:
Thing is, we never agreed on guidelines. I have proposed to @jmirtsch to use something visible instead of two spaces and we found the backslash option. The change is coming from @czapplitec though, not me. 😄